src/Altinn.Platform/Altinn.Platform.Storage/Storage/Helpers/DataElementHelper.cs

```diff
@@ -0,0 +1,67 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Altinn.Platform.Storage.Models;
+using Storage.Interface.Models;
+
+namespace Altinn.Platform.Storage.Helpers
+{
+    /// <summary>
+    /// DataElement helper methods
+    /// </summary>
+    public static class DataElementHelper
+    {
+        private static readonly string Prefix = "storage/api/v1";
```

Comment:
Data element helper should not have this prefix. It should be ignorant of the path/versionnumber of the api. Remove prefix and the stuff about selfLinks.

Reply:
Understood, fixed now.